packages/core/components/ExternalInput/index.tsx

```diff
@@ -147,12 +191,14 @@ export const ExternalInput = ({
                 })}
               </tbody>
             </table>
-          </div>
 
-          <div className={getClassNameModal("noContentBanner")}>No content</div>
+            <div className={getClassNameModal("loadingBanner")}>
+              <ClipLoader size={24} />
```

Comment:
This needs an `aria-label="Loading"`

Reply:
Addressed all of these comments. Also added a new commit to fix some style leakage in Nextra.